index.bs

```diff
@@ -1901,8 +1913,8 @@ or it MAY decide to accept the registration, e.g. while deleting the older regis
 
 ## Verifying an authentication assertion ## {#verifying-assertion}
 
-When verifying a given {{PublicKeyCredential}} structure (|credential|) as part of an [=authentication=] [=ceremony=], the [RP]
-MUST proceed as follows:
+When verifying a given {{PublicKeyCredential}} structure (|credential|) with PublicKeyCredentialType <a>public-key</a> 
```

Comment:
Again, format this decision as part of the algorithm. `If |credential|.type is: <dl class="switch"><dt>"public-key"</dt> …`